python/pylibraft/pylibraft/distance/fused_l2_nn.pyx

```diff
@@ -88,7 +91,15 @@ def fused_l2_nn_argmin(X, Y, output, sqrt=True):
                                       dtype=cp.float32)
         output = cp.empty((n_samples, 1), dtype=cp.int32)
 
-        fused_l2_nn_argmin(in1, in2, output)
+        # A single RAFT handle can optionally be reused across
```

Comment:
Quick comment: we might want to adopt a docstring generator like the one in cuML for common ones like this

Reply:
Yes I think centralizing this is a great idea. I'll start looking into the doc function in cuml